docs/release-notes/4142-tensorboard-profiler-data-from-workers.txt

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+**New Features**
+
+-  TensorBoard:
+      A new method, `TrainContext.upload_tensorboard_files()`, copies TensorBoard profiling data
+      files to checkpoint storage while optionally renaming them. It enables uploading profiling
+      data from non-chief workers (TensorBoard metrics and profiling data from chief is uploaded
+      automatically).
```

Comment:
This looks good, but is going to need additional explanation that:
- the call is only exposed to Core API users
- Trial API uses this new call under the hood to ship profiling data from all workers